src/base/bittorrent/session.cpp

```diff
@@ -1136,29 +1139,32 @@ void Session::configure(libtorrent::settings_pack &settingsPack)
     settingsPack.set_int(libt::settings_pack::outgoing_port, outgoingPortsMin());
     settingsPack.set_int(libt::settings_pack::num_outgoing_ports, outgoingPortsMax() - outgoingPortsMin() + 1);
 
-    // Ignore limits on LAN
-    settingsPack.set_bool(libt::settings_pack::ignore_limits_on_local_network, ignoreLimitsOnLAN());
     // Include overhead in transfer limits
     settingsPack.set_bool(libt::settings_pack::rate_limit_ip_overhead, includeOverheadInLimits());
     // IP address to announce to trackers
     settingsPack.set_str(libt::settings_pack::announce_ip, announceIP().toStdString());
     // Super seeding
     settingsPack.set_bool(libt::settings_pack::strict_super_seeding, isSuperSeedingEnabled());
-    // * Max Half-open connections
-    settingsPack.set_int(libt::settings_pack::half_open_limit, maxHalfOpenConnections());
     // * Max connections limit
     settingsPack.set_int(libt::settings_pack::connections_limit, maxConnections());
     // * Global max upload slots
     settingsPack.set_int(libt::settings_pack::unchoke_slots_limit, maxUploads());
     // uTP
     settingsPack.set_bool(libt::settings_pack::enable_incoming_utp, isUTPEnabled());
     settingsPack.set_bool(libt::settings_pack::enable_outgoing_utp, isUTPEnabled());
-    // uTP rate limiting
-    settingsPack.set_bool(libt::settings_pack::rate_limit_utp, isUTPRateLimited());
     settingsPack.set_int(libt::settings_pack::mixed_mode_algorithm, isUTPRateLimited()
                          ? libt::settings_pack::prefer_tcp
                          : libt::settings_pack::peer_proportional);
 
+#if LIBTORRENT_VERSION_NUM < 10100
```

Comment:
You're inside a bigger ifdef. This code is libtorrent 1.1.x only. Libtorrent 1.0.x doesn't have settings_pack anyway.